client-v2/src/components/FrontsCAPIInterface/FilterItem.tsx

```diff
@@ -25,7 +25,7 @@ const FilterItem = ({ children, onClear }: FilterItemProps) => (
   <SearchTermItem>
     {children}
     <SmallRoundButton onClick={() => onClear()} title="Clear search">
-      <ClearButtonIcon src={moreImage} alt="" height="22px" width="22px" />
+      <ClearIcon size={22} />
```

Comment:
minor: I wonder if `size` could be `small`, `normal`, `big` to reduce the amount of possible variance?

Reply:
There are 6 different Icon sizes in use but I can do something like the map below? I think this is because the icons come from different sources they scale differently. 
```
const SizeMap = {
  xs: 10,
  s: 12,
  m: 14,
  l: 22,
  xl: 30,
  xxl: 40
};
```